tests/functional/test_feeds.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+
+@pytest.mark.functional
```

Comment:
Curious: what does this mean/do?

Reply:
It is just a way of attaching a tag or label to the test function. See https://docs.pytest.org/en/latest/mark.html

There are some built-in markers (eg. `pytest.mark.parametrize`) which have special handling.

Reply:
That was my hunch. I had grepped our code before when working on functional tests and also done some Googling and could tell there was no "builtin" meaning to this tag. So I assumed that's what it was "for" but wasn't sure if any of our infrastructure made actual use of it. Thanks!